productivity/tests/test_driver.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+
+from productivity.mock import ProductivityPLC
+
+
+@pytest.fixture
+def plc_driver():
+    return ProductivityPLC('fake ip', 'tests/plc_tags.csv')
+
+
+def test_init():
+    with pytest.raises(TypeError, match='unsupported data type'):
+        ProductivityPLC('fake ip', 'tests/bad_tags.csv')
+
+
+def test_get_tags(plc_driver):
+    expected = {
+        'AV-101': {'address': {'start': 1, 'end': 1}, 'id': 'DO-0.1.3.1',
```

Comment:
These are just tests so okay to pull. But we should rename these in a future commit to match snake_case av101, toggle_av101, etc naming.